docs/mvvm/source-gen/diagnostics/MVVMTK0008.md

```diff
@@ -0,0 +1,35 @@
+---
+title: MVVM Toolkit error MVVMTK0008
+author: Sergio0694
+description: MVVM Toolkit error MVVMTK0008
+keywords: community toolkit, dotnet, csharp, mvvm, net core, net standard, source generators
+dev_langs:
+  - csharp
+---
+
+# MVVM Toolkit error MVVMTK0008
+
+The source generator features from the MVVM Toolkit require consuming projects to set the C# language version to at least C# 8.0. Make sure to add `<LangVersion>8.0</LangVersion>` (or above) to your .csproj file.
+
+The following sample generates MVVMTK0008:
+
+```xml
+<PropertyGroup>
+    <LangVersion>7.3</PropertyGroup>
+</PropertyGroup>
+```
+```csharp
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace MyApp;
+
+// The project doesn't have a high enough version of C#
+[INotifyPropertyChanged]
+public partial class SampleViewModel
+{
+}
+```
+
+## Additional resources
+
+- You can find more examples in the [unit tests](https://github.com/CommunityToolkit/dotnet/tree/main/tests/CommunityToolkit.Mvvm.SourceGenerators.UnitTests).
```

Comment:
Link to your blog maybe about LangVersion as that question gets asked a lot?

Reply:
Good idea! Done in https://github.com/MicrosoftDocs/CommunityToolkit/pull/113/commits/0718566f0b8505037dc58474ab06eb8b2e4efa96 🙂